common/static/common/js/discussion/utils.js

```diff
@@ -446,17 +473,23 @@
 
         DiscussionUtil.abbreviateHTML = function(html, minLength) {
             var $result, imagesToReplace, truncated_text;
-            truncated_text = jQuery.truncate(html, {
+            truncated_text = edx.HtmlUtils.HTML(jQuery.truncate(html, {
```

Comment:
Is `html` really HTML?  If so, could it be set to an HTML Snippet using `HtmlUtils` before being passed to this function?  To use `JQuery.truncate()`, you might then need to leave this line as-is and add `html.toString()`, but at least you would have some context for declaring this as HTML().  

If html is really plain text, you could rename it and just drop this call to `HtmlUtils.HTML()`.